controllers/handlers_utils.go

```diff
@@ -936,6 +963,7 @@ func deployObjects(ctx context.Context, deployingToMgmtCluster bool, destClient
 
 	for i := range referencedObjects {
 		var tmpResourceReports []configv1beta1.ResourceReport
+		subressources := getSubressources(referencedObjects[i], logger)
```

Comment:
For ConfigMap and Secret we get sub resources inside deployContentOfConfigMap and deployContentOfSecret. 

instead of getting sub resources here, can we get sub resource also inside deployContentOfSource?

So it is same approach for all.